src/utils.cpp

```diff
@@ -82,6 +82,19 @@ std::string IR2Vec::getDemagledName(const llvm::Function *function) {
   return demangledName;
 }
 
+// Function to get demanged instruction name
+std::string IR2Vec::getDemagledName(const llvm::Instruction *instruction) {
```

Comment:
Why is this method needed?

Reply:
The `refactoredCore.cpp` has the method, `createInstructionVectorDict` . The method had duplicated code to get the `Demangled` Name for the instruction. So, in interest of better modularity, and reusability, I added this function here.

Reply:
What does demangling instruction mean?

Reply:
The code was used to get the `demangled`  format name for the instruction. I shifted that code to the include files to improve modularity